docs/details/pmm_components_and_versions.md

```diff
@@ -0,0 +1,21 @@
+# PMM components and versions
+
+The following table lists all the PMM client/server components and their versions:
+
+
+| Component| Version| URL                                           | GitHub URL                                                                                                  |
```

Comment:
```suggestion
| Component| Version| Documentation                                 | Repository                                                                                                  |
```

Reply:
@rasika-chivate It feels like we still call fill in the blank cells in the table. I have a number of suggestions, let's discuss.

<img width="1159" alt="Screenshot 2022-09-21 at 10 58 11" src="https://user-images.githubusercontent.com/81549/191449317-d5f7b02e-fc2a-46fc-a381-b54d2ae5f349.png">